PocketBeagle/Grove/KeyBoard_Player.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) 2020 SeeedStudio
```

Comment:
Why is this rewritten in the wrong directory? The way all of this is done, it makes it impossible for me to actually look for changes using version control.